src/start_server.ts

```diff
@@ -150,7 +155,40 @@ export function getApp(options: ServerOptions): express.Express {
 
   app.use('/components/', polyserve);
 
+  if (options.proxy) {
+    if (options.proxy.path.startsWith('components')) {
+      console.error('proxy path can not start with components.');
+      return;
+    }
+
+    let escapedPath = '';
+    let escaped = false;
+    for (const char of options.proxy.path) {
+      escaped = (char === '\\') ? !escaped : false;
+      if (!escaped && (char === '*' || char === '+' || char === '?')) {
+        escapedPath += '\\\\';
```

Comment:
I had thought the same. If these are 2 backslashes, the following syntax errors are thrown when compiling the RegExp for the Express path:

```
  1) startServer proxy escapes path with regex symbols:
     Error: SyntaxError: Invalid regular expression: /^\/\+regex\(.*)path\?\/?(?=\/|$)/: Unmatched ')'
      at lib/start_server.js:77:19
      at next (native)
      at fulfilled (lib/start_server.js:17:58)

  2) startServer proxy does not unescape escape symbols in path:
     Error: SyntaxError: Invalid regular expression: /^\/\+regex\?path\(.*)\/?(?=\/|$)/: Unmatched ')'
      at lib/start_server.js:77:19
      at next (native)
      at fulfilled (lib/start_server.js:17:58)
```

It seems that express simply looks for `*` in the route and then replaces, without properly escaping. The underlying path matching of `express` uses `path-to-regexp` which has this comment in the code: https://github.com/pillarjs/path-to-regexp/blob/master/index.js#L18-L20

E.g. if you want to escape special characters, double backslashes should be used.

I have tried to use 2 backslashes for everything except the express path, but that was unsuccessful too.